modules/ROOT/pages/storage.adoc

```diff
@@ -175,6 +175,36 @@ storage:
 
 NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
 
+== Sizing the root partition
+
+Currently, as soon as you reconfigure or move the root partition, it is no longer automatically grown on first boot (see related discussions in https://github.com/coreos/fedora-coreos-tracker/issues/570[this issue]). In the case of moving the root partition to a new disk (or multiple disks), you should be setting the desired partition size(s) as part of your config (via `size_mib`). If reconfiguring the root filesystem in place (e.g. as in the LUKS example above), you can tell Ignition to grow the existing partition to take up the whole disk like this:
+
+.Resizing the root partition to its maximum size
+[source.yaml]
+----
+variant: fcos
+version: 1.2.0
+storage:
+  disks:
+    - device: /dev/vda
+      partitions:
+        - label: root
+          number: 4
```

Comment:
Should be redundant.

Reply:
Without it, Ignition creates another partition with label `root`. (Technically, `label: root` is not necessary, but I chose to include it for clarity.)

Reply:
Whoops, you are correct.  I misread the spec.